build.gradle

```diff
@@ -338,7 +338,30 @@ subprojects {
     classpath = sourceSets.integrationTest.runtimeClasspath
     outputs.upToDateWhen { false }
 
-    jvmArgs = ['-Xms512m', '-Xmx1g']
+    jvmArgs = [
+      '-Xms512m',
+      '-Xmx1g',
+      '--add-opens',
+      'java.base/sun.security.provider=ALL-UNNAMED',
+      // Mockito and jackson-databind do some strange reflection during tests.
+      // This suppresses an illegal access warning.
+      '--add-opens',
```

Comment:
this appears more than requested by the OR-335 ticket - do we need all of this?

Reply:
updated the comment. because it's same but different